components/CommonComponents/ShellBox/index.module.scss

```
@@ -0,0 +1,75 @@
+.shell-box {
```

I believe this should be `shellBox`, as it is used here

https://github.com/nodejs/nodejs.org/pull/5234/files#diff-a315b5aa06fa6054f4a1a831400da67bd944d83eda8da6aa7484783b0b9be4dbR19

```suggestion
.shellBox {
```

---

I don't understand how it's work for nodejs.dev 🤔

---

CSS modules natively translate hyphen classes to camelCase. I think it is done by Next already https://github.com/vercel/next.js/discussions/11267